tests/test_builders/test_build_html_copyright.py

```diff
@@ -9,6 +9,13 @@
 LOCALTIME_2009 = type(LT)(LT_NEW)
 
 
+@pytest.fixture
+def source_date_empty(monkeypatch):
+    with monkeypatch.context() as m:
+        m.delenv('SOURCE_DATE_EPOCH', raising=False)
+        yield
```

Comment:
If we only use this once, is it worth being a standalone function? 

If we should keep it, could we add a docstring to describe why and when we should use the fixture?

A

Reply:
When I experimented with this, I initially tried inlining it into the relevant test, but that didn't work since the monkeypatch needs to be applied before the `app` fixture is started.